<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FavoritesOverBtns</class>
 <widget class="QWidget" name="FavoritesOverBtns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="hspcCenter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="btnEndEdit">
     <property name="toolTip">
      <string>Finish editing</string>
     </property>
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/ok.png</normaloff>:/pic/ok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Edit favorite commands</string>
     </property>
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/lstbtnedit.png</normaloff>:/pic/lstbtnedit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnDelete">
     <property name="toolTip">
      <string>Delete this favorite command</string>
     </property>
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/lstbtnremove.png</normaloff>:/pic/lstbtnremove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="hspcLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnInsert">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Insert this favorite command at current position in config</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>insert</string>
     </property>
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/rightarrow.png</normaloff>:/pic/rightarrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>16</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bibolamazi_res.qrc"/>
 </resources>
 <connections/>
</ui>
